add.go

```diff
@@ -79,11 +93,14 @@ func (b *Builder) Add(destination string, extract bool, source ...string) error
 			return fmt.Errorf("error reading %q: %v", src, err)
 		}
 		if fi.Mode().IsDir() {
```

Comment:
Ok, personal twitch here.  I'm not at all a fan of one or two character variable names in code, especially so when the short name is part of a regular word like 'fi', and even more so when the variable is used more than 4 or 5 lines away from where it's first declared.  I much prefer longer/clearer variable names as it makes long term maintenance much easier.  I'll now crawl under my rock.